linebot/imagemap.go

```diff
@@ -71,31 +72,36 @@ type URIImagemapAction struct {
 func (a *URIImagemapAction) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&struct {
 		Type    ImagemapActionType `json:"type"`
+		Label   string             `json:"label"`
```

Comment:
The document says this field is optional. Can we include "omitempty" in this tag just like other Actions such as `PostBackAction`, `URIAction`, etc?